pkg/controllers/dashboard/controller.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/rancher/rancher/pkg/controllers/dashboard/clusterindex"
 	"github.com/rancher/rancher/pkg/controllers/dashboard/fleetcharts"
 	"github.com/rancher/rancher/pkg/controllers/dashboard/helm"
+	"github.com/rancher/rancher/pkg/controllers/dashboard/hostedkubernetecharts"
```

Comment:
Is this spelling intentional?

Reply:
Resolving due to https://github.com/rancher/rancher/pull/33004#discussion_r652315459